server/src/main/java/org/opensearch/indices/cluster/IndicesClusterStateService.java

```diff
@@ -538,7 +538,7 @@ private void createIndices(final ClusterState state) {
         for (Map.Entry<Index, List<ShardRouting>> entry : indicesToCreate.entrySet()) {
             final Index index = entry.getKey();
             final IndexMetadata indexMetadata = state.metadata().index(index);
-            logger.debug("[{}] creating index", index);
+            logger.info("[{}] creating index", index);
```

Comment:
Why do you need this to be info?

Reply:
Sorry, was an unintended change. Removed now.